src/session.rs

```diff
@@ -885,6 +885,12 @@ impl Session {
         self.edit_view(id);
     }
 
+    pub fn as_blank(mut self, fs: FileStatus, w: u32, h: u32) -> std::io::Result<Self> {
```

Comment:
You can just return `Self`, because this function cannot fail. Then you can remove the `?` in the invocation.